scripts/sct_label_vertebrae.py

```diff
@@ -75,9 +76,9 @@ def get_parser():
                       example="t2_seg.nii.gz")
     parser.add_option(name="-c",
                       type_value="multiple_choice",
-                      description="type of image contrast, t2: cord dark / CSF bright ; t1: cord bright / CSF dark",
-                      mandatory=True,
-                      example=['t1', 't2'])
+                      description="type of image contrast, automatically detected but can be specified manually (t2: cord dark / CSF bright ; t1: cord bright / CSF dark)",
+                      mandatory=False,
+                      example=['t1', 't2', 't2s'])
```

Comment:
Same comment as for `sct_detect_pmj`.

Reply:
https://github.com/neuropoly/spinalcordtoolbox/pull/2328#issuecomment-508770873